content/en/docs/concepts/security/windows-pod-security-profile.md

```diff
@@ -0,0 +1,274 @@
+---
+reviewers:
+- tallclair
+title: Windows Pod Security Policy
+description: >
+  Clarification of which pod security policies apply to Windows pods
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Windows in Kubernetes has some differentiators from standard Linux-based workloads. Many 
+of the Pod SecurityContext fields [have no effect on
+Windows](/docs/setup/production-environment/windows/intro-windows-in-kubernetes/
+#v1-podsecuritycontext). Windows HostProcess containers also differ from traditional 
+privileged containers, which causes some additional policies to lose their applicability 
+for Windows.
+
+This guide outlines the applicable _policies_ for Windows Pod Security Standards and 
+redefines the [existing profiles](/docs/concepts/security/pod-security-standards) for the 
+Windows context.
```

Comment:
I was not expecting a second page redefining pod security standards... this will be very difficult to keep in sync, and I expect will lead to confusion. Once there's a standard way to determine a pod will only run on a windows node, I expected the main pod security standards page to be updated indicating which items do not need to be enforced for windows pods

Reply:
> Once there's a standard way to determine a pod will only run on a windows node, I expected the main pod security standards page to be updated indicating which items do not need to be enforced for windows pods.

That makes sense to me too.

Reply:
@liggitt Looking at https://github.com/kubernetes/website/blob/dev-1.22/content/en/docs/concepts/security/pod-security-standards.md most of the items do not make sense for windows. 

For 1.22 should we add a ## section for windows support and explicitly name the items that are supported by Windows or should we update all the items to indicate they don't need to be enforced by windows.

I think just the following are 
- hostProcess
- hostNetwork
- runAsNonRoot
- volumes (of most types)

Reply:
@brasmith-ms perhaps we should update https://github.com/kubernetes/website/blob/dev-1.22/content/en/docs/concepts/security/pod-security-standards.md#what-profiles-should-i-apply-to-my-windows-pods to call out the differences for Windows?

Reply:
> For 1.22 should we add a ## section for windows support and explicitly name the items that are supported by Windows or should we update all the items to indicate they don't need to be enforced by windows.

I don't think so. Pod Security Standards are enforced on Pod API objects, not by the node. Once there's a way for things looking at the API object to determine definitively it is a windows pod, *then* we can update the pod security standards page to indicate which checks apply to windows pods.

Reply:
@liggitt are you saying we should not update the pod security standards page for 1.22 to account for HostProcess containers? The hostProcess field is available in 1.22 and should be documented somewhere, no?

Reply:
From the user perspective I think it's important to know which security standards apply to Windows nodes when attempting to set up a HostProcess pod

Reply:
> @liggitt are you saying we should not update the pod security standards page for 1.22 to account for HostProcess containers? The hostProcess field is available in 1.22 and should be documented somewhere, no?

No, I'm saying we *should* add it to the pod security standards page and indicate it may not be set to true in the baseline policy.

We should *not* add a second windows-only copy of the pod security standards page, or start annotating the pod security standards checks with which OS they apply to yet.

> From the user perspective I think it's important to know which security standards apply to Windows nodes when attempting to set up a HostProcess pod

The Pod Security Standards apply to the shape of the Pod API objects. Once https://github.com/kubernetes/enhancements/pull/2803 is complete, we can update the Pod Security Standards page to take that into account.

Reply:
@liggitt Okay got it, thanks for the insight. I'll update this page now.

Reply:
Hi @liggitt, I have updated with the changes mentioned here. Is this more reasonable?